courses/course-planning/degree-requirements.md

```diff
@@ -0,0 +1,42 @@
+---
+id: degree-requirements
+title: Degree Requirements
+sidebar_label: Degree Requirements
+slug: /course-planning/degree-requirements
+---
+
+# Degree Requirements
+
```

Comment:
Add a `_Last updated on {date}_` line at the top along along with a disclaimer somewhere near the top warning that the requirements are subject to change and students are advised to refer to the university's official calendar for the latest updates.

Format for a warning:

```
:::warning
Warning content
:::
```